code/game/machinery/computer/law.dm

```diff
@@ -1,3 +1,5 @@
+#define EMAG_COOLDOWN 60 SECONDS
```

Comment:
```suggestion
#define AIUPLOAD_EMAG_COOLDOWN 60 SECONDS
```
should use a more specific define name even though we're undef'ing it, improves code readability and potential future confusion about why this is scoped to this file

Reply:
applied